tokio-console/src/config.rs

```diff
@@ -48,6 +50,26 @@ pub struct Config {
     #[clap(long = "log", env = "RUST_LOG")]
     log_filter: Option<LogFilter>,
 
+    /// Enable lint warnings.
+    ///
+    /// This is a comma-separated list of warnings to enable.
+    ///
+    /// Each warning is specified by its name, which is one of:
+    ///
+    /// * `self-wakes` -- Warns when a task wakes itself more than a certain percentage of its total wakeups.
+    ///
+    /// * `lost-waker` -- Warns when a task is dropped without being woken.
+    ///
+    /// * `never-yielded` -- Warns when a task has never yielded.
+    ///
+    #[clap(long = "warn", short = 'W', value_delimiter = ',', num_args = 1..)]
```

Comment:
if i understand correctly, the use of `num_args = 1..` mean that there's currently no way to disable _all_ warnings from the CLI currently? since passing `--warn` with no warning names is invalid, and by default we enable all warnins.

Reply:
As we discussed on Discord, we will add the `--Allow` flag to disable warnings. @hds proposed adding the `ALL` value to `--Allow` to disable all warnings.